policyengine_us/parameters/gov/states/md/tax/income/agi/subtractions/hundred_year/age_threshold.yaml

```diff
@@ -0,0 +1,9 @@
+description: Maryland allows filers at or above this age to receive the hundred year subtraction. 
+values:
+  2022-01-01: 100
+metadata:
+  reference:
+    - title: MD 2022 Resident Booklet
+      href: https://www.marylandtaxes.gov/forms/22_forms/Resident_Booklet.pdf#page=12
```

Comment:
https://casetext.com/statute/code-of-maryland/article-tax-general/title-10-income-tax/subtitle-2-maryland-taxable-income-calculations-for-individual/part-ii-maryland-adjusted-gross-income/section-10-207-subtractions-from-federal-adjusted-gross-income-in-general (nn)